include/deal.II/dofs/dof_tools.h

```diff
@@ -2385,6 +2385,27 @@ namespace DoFTools
    * @image html support_point_dofs1.png
    * @image html support_point_dofs2.png
    * </p>
+   *
+   * To generate the mesh and the support point info in a single gnuplot file,
+   * use code similar to
+   * @code
+   * std::ofstream out("gnuplot.gpl");
+   * out << "plot '-' using 1:2 with lines, '-' with labels point pt 2 offset 1,1" << std::endl;
+   * GridOut().write_gnuplot (triangulation, out);
+   * out << "e" << std::endl;
```

Comment:
Hm, how interesting -- when you do `plot '-'` it takes data from the command line enclosed between `e...e`? Does it have to be `e`? `e` is not a particularly memorable delimiter since it also appears inside numbers like `1.234e5`.